src/odemis/driver/xt_client.py

```diff
@@ -358,6 +358,16 @@ def check_and_transfer_latest_package(self) -> None:
                     update.show()
                 else:
                     logging.warning("{} is a bad file in {} not transferring latest package.".format(ret, package.path))
+
+                # This version supports reading raw stage coordinates
+                if current_version is not None and bitness is not None:
```

Comment:
Don't check explicitly for the version. Instead, I would suggest that if (and *only if*) the raw coordinates are requested, and the call to `set_raw_coordinate_system()`  fails *because the method doesn't exist*, then you raise a ValueError (or maybe, more appropriately, a NotImplementedError).

So, in particular, if the user asks raw_coordinates = False, and calling the `set_raw_coordinate_system(False)` fails, it should continue business as usual.